.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
 composer.lock
+.idea
```

Comment:
Please don't add your ide's specific files here, configure a global `.gitignore` to do that.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Good advice, I did not know about this 👍